pytrnsys_process/plotting/plotters.py

```diff
@@ -211,7 +210,7 @@ def _do_plot(
         df: _pd.DataFrame,
         columns: list[str],
         use_legend: bool = True,
```

Comment:
This could be adjusted:
- kick out and leave up to the User?
- ChartBase(use_legend=True/False).maybe_legend()
    - ChartBase.set_legend_stuff(names, fontsize, )